### PR TITLE
added special booking values

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: HappyTravel.EdoContracts/Accommodations/Booking.cs [hunk 1]
```diff
@@ -69,5 +71,10 @@ public static Booking Empty(string referenceCode, BookingStatusCodes statusCode,
         ///     The reference code obtained from a supplier.
         /// </summary>
         public string SupplierReferenceCode { get; }
+        
+        /// <summary>
+        ///     Special booking values
+        /// </summary>
+        public List<KeyValuePair<string, string>> SpecialBookingValues { get; }
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/Accommodations/Booking.cs:78]: Just "SpecialValues" I think, these are already inside the booking.

File: HappyTravel.EdoContracts/Accommodations/Booking.cs [hunk 2]
```diff
@@ -69,5 +71,10 @@ public static Booking Empty(string referenceCode, BookingStatusCodes statusCode,
         ///     The reference code obtained from a supplier.
         /// </summary>
         public string SupplierReferenceCode { get; }
+        
+        /// <summary>
+        ///     Special booking values
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/Accommodations/Booking.cs:76]: Let's add more information for the ancestors, like:
"Used for additional information about the booking to show in voucher and other places"

File: HappyTravel.EdoContracts/Accommodations/Booking.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Booking.cs]: Fixed

File: HappyTravel.EdoContracts/Accommodations/Booking.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.EdoContracts/Accommodations/Booking.cs]: Fixed

**Review by mingazhev — APPROVED** [review]

**Pull request merged**